converters/fromZigbee.js

```diff
@@ -5300,31 +5312,6 @@ const converters = {
             return payload;
         },
     },
-    xiaomi_battery: {
-        cluster: 'genBasic',
-        type: ['attributeReport', 'readResponse'],
-        convert: (model, msg, publish, options, meta) => {
-            let voltage = null;
-            if (msg.data['65281']) {
-                voltage = msg.data['65281']['1'];
-            } else if (msg.data['65282']) {
-                voltage = msg.data['65282']['1'].elmVal;
-            }
-
-            if (voltage) {
-                const payload = {
-                    voltage: voltage, // @deprecated
-                    // voltage: voltage / 1000.0,
-                };
-
-                if (model.meta && model.meta.battery && model.meta.battery.voltageToPercentage) {
```

Comment:
This logic is missing https://github.com/Koenkk/zigbee-herdsman-converters/blob/530b629569428661ccf5cc207a4ac490116648a5/lib/xiaomi.js#L412

Reply:
Yes, you are right. For others elements, when no defined, we use `batteryVoltageToPercentage(value, '3V_2100');` as default. Do you want that I add it here or you prefer to simply not declare percentage?

Reply:
If no `meta.voltageToPercentage` is defined, no voltage -> battery % converting should take place.

Reply:
In the old `aqara_opple` converter, for the value "1" it used it directly:

https://github.com/Koenkk/zigbee-herdsman-converters/blob/13f1160bb2a5406f7de5ae773d17eae03c371299/converters/fromZigbee.js#L5371-L5373

I saw other converters that use the 1 the with the meta, so in the new utility file, , I use the `meta.voltageToPercentage` if defined and if not I use the old conversion directly.

Do you want that I remove it from here too then? 
https://github.com/Koenkk/zigbee-herdsman-converters/blob/9427a03d3d53f198a5c9c22ec9ceaa856693eca5/lib/xiaomi.js#L151-L154

I think is cleaner. The worst thing that can happen is that some device that don't define the meta will not have now a percentage, and is easy to fix adding the correct meta.

Reply:
Yes, please. Only when `meta.voltageToPercentage` is defined, voltage -> battery % calculation should be done.

Reply:
Fixed!